esp/bindings/http/client/httpclient.cpp

```diff
@@ -222,8 +226,8 @@ int CHttpClient::connect(StringBuffer& errmsg)
 
     try
     {
-        if(m_timeout)
-            m_socket = ISocket::connect_timeout(ep,m_timeout);
+        if(connectTimeoutMs)
```

Comment:
I almost did that, but erred on the side of only ever affecting underlying behavior if the caller explicitly requested it.  I'll think about it again.